src/classic/clvm/serialize.rs

```diff
@@ -138,7 +135,7 @@ pub trait OpStackEntry {
 
 type TOpStack<'a> = Vec<Option<Box<dyn OpStackEntry>>>;
 
-pub fn sexp_to_stream<'a>(allocator: &'a mut Allocator, sexp: NodePtr, f: &mut Stream) {
```

Comment:
we already have serialization functions in `clvm_rs`. they look very similar to these